drafter.gyp

```diff
@@ -41,17 +41,33 @@
         "src/RefractAST.h",
         "src/RefractAST.cc",
 
+        # librefract parts - will be separated into other project
+
         "src/refract/Element.h",
         "src/refract/Element.cc",
         "src/refract/Typelist.h",
         "src/refract/VisitableBy.h",
+
         "src/refract/Visitor.h",
+        "src/refract/Visitors.h",
+
         "src/refract/SerializeCompactVisitor.h",
         "src/refract/SerializeCompactVisitor.cc",
         "src/refract/SerializeVisitor.h",
         "src/refract/SerializeVisitor.cc",
         "src/refract/ComparableVisitor.h",
+        "src/refract/ComparableVisitor.cc",
+        "src/refract/TypeQueryVisitor.h",
+        "src/refract/TypeQueryVisitor.cc",
+        "src/refract/IsExpandableVisitor.h",
+        "src/refract/IsExpandableVisitor.cc",
+        "src/refract/ExpandVisitor.h",
```

Comment:
Basically, This `ExpandVisitor` file is about expanding the named types given other named types right? I think that should be in drafter.

The refract spec only defines the `extend` stuff.